README.md

```diff
@@ -220,16 +218,11 @@ See in file [REFERENCE.md](REFERENCE.md).
 
 ## Limitations
 
-- Currently tested manually on Centos 7, but we will eventually add automated
-  testing and are targeting compatibility with other platforms.
-- Tested with Puppet 5.x
+- Tested with Splunk 7.2.4.2, 9.0.0, and 9.2.0.1. No official support for Splunk 8.
+- Tested with Puppet 7 and Puppet 8.
```

Comment:
These match what's in `metadata.json`. Should we leave it out, just like we don't list the operating systems from there?

Reply:
I guess we can safely remove this yeah, people can look at this on puppet forge 👍